src/ray/core_worker/reference_count.h

```diff
@@ -44,8 +55,14 @@ class ReferenceCounter {
   /// dependencies are inlined or when the task finishes for plasma dependencies.
   ///
   /// \param[in] object_ids The object IDs to remove references for.
+  /// \param[in] worker_addr The address of the worker that executed the task.
+  /// \param[in] borrowed_refs The references that the worker borrowed during
+  /// the task. Some references in this table may still be borrowed by the
+  /// worker and/or a task that the worker submitted.
   /// \param[out] deleted The object IDs whos reference counts reached zero.
   void RemoveSubmittedTaskReferences(const std::vector<ObjectID> &object_ids,
```

Comment:
`UpdateFinishedTaskReferences`?

Reply:
Going to change it to `UpdateSubmittedTaskReferences` for now. Will split into 2 methods in a future PR, as mentioned above.